### ISSUE TITLE
NTR: Bordetella pertussis PCR in nasopharyngeal secretions positive (Lab: bacteriology, virology, parasitology, fungal)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8088]
New term request Bordetella pertussis PCR in nasopharyngeal secretions positive (nadea3_190213031414): Polymerase chain reaction; DNA

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000962
- label: Positive respiratory tract Bordetella pertussis nucleic acid test
- parents: [Positive respiratory tract nucleic acid pathogen test (HP:6000836)](https://hpo.jax.org/app/browse/term/HP:6000836)
- pmid: 
- def: Detection of Bordetella pertussis PCR nucleic acid in in nasopharyngeal secretions by a method such as polymerase chain reaction.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8088
- synonyms:
